environments/staging/inventory.ini

```diff
@@ -70,8 +70,11 @@ hqdb1
 # Amazon EC2
 # celery0
 
+[hqpillow0]
+hqpillow0-staging.internal-va.commcarehq.org
+
 [pillow0]
-10.201.10.41 hostname=pillow0-staging ec2=yes
+# 10.201.10.41 hostname=pillow0-staging ec2=yes
```

Comment:
I wouldn't comment this out. In order to deploy to it (with no pillows on it), the machine has to exist

Reply:
^^ noted. Is there any action or shall I merge this in to get staging out of its current weird state?

Reply:
nope, this is good